rpmconf/Install/Util/utilfunc.sh

```diff
@@ -2317,18 +2317,35 @@ getInstallPackages() {
 
     # askInstallPkgYN args : PROMPT REQUIRE_STORE=yes|no YES_STORE_DEFAULT=Y|N NO_STORE_DEFAULT=Y|N
 
+    ZIMBRAINTERNAL=no
+    echo $HOSTNAME | egrep -qe 'eng.zimbra.com$|lab.zimbra.com$|zimbradev.com$' > /dev/null 2>&1
+    if [ $? = 0 ]; then
+       ZIMBRAINTERNAL=yes
+    fi
     if [ $i = "zimbra-license-tools" ]; then
       response="yes"
     elif [ $i = "zimbra-modern-ui" ]; then
       ifStoreSelectedY
     elif [ $i = "zimbra-modern-zimlets" ]; then
       ifStoreSelectedY
     elif [ $i = "zimbra-patch" ]; then
-      ifStoreSelectedY
+	    if [ x"$ZIMBRAINTERNAL" = "xyes" ] && [ $STORE_SELECTED = "yes" ]; then
+		    askYN "Install $i" "Y"
```

Comment:
indentation is not consistent with other existing if blocks, would be better to keep it consistent.